backend/src/main/java/ca/bc/gov/hlth/hnweb/controller/MspContractsController.java

```diff
@@ -75,7 +75,11 @@
 	
 	 /**
 	 * Get MSP Coverage info for a Personal Health Number (PHN) of a group Inquiry
-	 * Maps to the legacy R40.
+	 * Maps to the legacy: R40.
```

Comment:
Javadoc formatting is now different between this method and others. Most don't use the colon.